src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -44,6 +44,17 @@ def _raise_error_if_not_drawing_classifier_input_sframe(
     if len(dataset) == 0:
         raise _ToolkitError("Input Dataset is empty!")
 
+def drop_missing_values(dataset, feature, is_train):
```

Comment:
This method should be hidden. Please prefix method name with a `_`.

Reply:
Okay